int16  id
string name
int8   status
